delta-lake/delta-33x/src/main/scala/com/nvidia/spark/rapids/delta/delta33x/Delta33xProvider.scala

```diff
@@ -91,6 +97,42 @@ object Delta33xProvider extends DeltaIOProvider {
     ).map(r => (r.getClassFor.asSubclass(classOf[RunnableCommand]), r)).toMap
   }
 
+  case class GpuIncrementMeticMeta(
+    cpuInc: IncrementMetric,
+    override val conf: RapidsConf,
+    p: Option[RapidsMeta[_, _, _]],
+    r: DataFromReplacementRule) extends ExprMeta[IncrementMetric](cpuInc, conf, p, r) {
+    override def convertToGpu(): GpuExpression = {
+      val gpuChild = childExprs.head.convertToGpu()
```

Comment:
To be defensive I would put a check in their to make sure it only has one child expression otherwise blow up. 

require(childExprs.length == 1)

Reply:
IncrementMetric is a unary expression, it is guaranteed